gopherage/cmd/html/static/parser.ts

```diff
@@ -126,6 +126,21 @@ export function parseCoverage(content: string): Coverage {
     throw new Error('Expected to start with mode line.');
   }
 
+  // Well-formed coverage files are already sorted alphabetically, but Kubernetes'
+  // `make test` produces ill-formed coverage files. This does actually matter, so
+  // sort it ourselves.
+  lines.sort((a, b) => {
+    a = a.split(':', 2)[0];
+    b = b.split(':', 2)[0];
```

Comment:
is it worth splitting on the last `:` or are we guaranteed to only have one? /shrug